Sources/UIScrollView+indicatorsInternal.swift

```diff
@@ -12,26 +12,55 @@
 internal extension UIScrollView {
     internal static let indicatorDistanceFromScrollViewFrame: CGFloat = 2.5
 
-    internal func indicatorOffsetsInContentSpace() -> (horizontal: CGFloat, vertical: CGFloat) {
-        let totalContentArea = (
-            horizontal: contentInset.left + contentSize.width + contentInset.right,
-            vertical: contentInset.top + contentSize.height + contentInset.bottom
-        )
+    internal var indicatorLengths: (horizontal: CGFloat, vertical: CGFloat) {
```

Comment:
clear about get{}, just slipped my mind that it's legal without it - correcting in the next commit.

controlled access levels and I was able to change a few to private, leaving only three functions that are indeed called from outside the file (from `UIScrollView.swift` in case of all 3)

Reply:
Access levels were still not corrected, must have not pushed some changes. pushed them now.